src/mlpack/core/data/map_policies/missing_policy.hpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file missing_policy.hpp
+ * @author Keon Kim
+ *
+ * Missing map policy for dataset info.
+ */
+#ifndef MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+#define MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+
+#include <mlpack/core.hpp>
+#include <unordered_map>
+#include <boost/bimap.hpp>
+#include <mlpack/core/data/map_policies/datatype.hpp>
+#include <limits>
+
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+/**
+ * Same as increment map policy, but does not change type of features.
+ */
+class MissingPolicy
+{
+ public:
+  // typedef of mapped_type
+  using mapped_type = double;
```

Comment:
To be consistent with the rest of the codebase please use camel casing for all names.